pysteps/utils/reprojection.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+from rasterio import Affine as A
+from rasterio.warp import reproject, Resampling
+import xarray as xr
+import numpy as np
+
+
+def reprojection(R_src, R_dst):
+    """Reprojects precipitation fields to the domain of another precipiation
```

Comment:
there are few typos in the docstrings
```suggestion
    """Reprojects precipitation fields to the domain of another precipitation
```